db/external_sst_file_ingestion_job.cc

```diff
@@ -633,7 +636,8 @@ Status ExternalSstFileIngestionJob::GetIngestedFileInfo(
     if (key.sequence != 0) {
       return Status::Corruption("External file has non zero sequence number");
     }
-    file_to_ingest->largest_internal_key.SetFrom(key);
+
+    file_to_ingest->largest_internal_key.Set(key.user_key, assigned_seqno, ValueType::kTypeValue);
```

Comment:
Same as above.

Reply:
same above, I could not find the key.type.